.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,38 +1,54 @@
 ---
 name: Bug Report
-about: Create a report to help us improve
-title: ''
-labels: 'Status: Needs Review, Type: Bug'
-assignees: ''
-
+about: Create a report to help us improve something that is not working correctly
+title: "Bug - :"
+labels: "Status: Needs Review, Type: Bug"
+assignees: ""
 ---
+
 <!-- Thank you for taking the time to submit a bug report to The Odin Project. In order to get issues closed in a reasonable amount of time, you must include a baseline of information about the bug in question. Please read this template in its entirety before filling it out to ensure that it is filled out correctly. -->
 
-<!-- Complete the following REQUIRED checkboxes by replacing the whitespace between the square brackets with an 'x', e.g. [x]. -->
-- [ ] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-- [ ] The title of this issue follows the `where the change occurs: brief description of bug` format, e.g. `React section: Knowledge Checks don't link to resource`
+Complete the following REQUIRED checkboxes:
+-   [ ] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
+-   [ ] The title of this issue follows the `Bug - location of bug: brief description of bug` format, e.g. `Bug - React section: Knowledge Checks don't link to resource`
+
+The following checkbox is OPTIONAL:
+<!-- Completing this checkbox does not guarantee you will be assigned this issue, but rather lets us know you are interested in working on it. -->
+-   [ ] I would like to be assigned this issue to work on it
 
-<!-- The following checkbox is OPTIONAL. Completing it does not guarantee you will be assigned this issue, but rather lets us know you are interested in working on it. -->
-- [ ] I would like to be assigned this issue to work on it
+<hr>
 
-**1. Description of the Bug**
+**1. Description of the Bug:**
 <!-- A clear and concise description of what the bug is. Include any screenshots that may help show the bug in action. -->
 
 
-**2. How To Reproduce**
-<!-- What steps one might need to take in order to reproduce this bug. -->
+**2. How To Reproduce:**
```

Comment:
I think "All steps to produce" would be a bit more clearer. I could imagine someone reading this and leaving out a detail thinking they can just give a generic overview. Think it would be a bit helpful if we made it explicit in case they ignore the commented out parts.

Reply:
For the desired behavior, I would say the Expected Behavior section covers that.

For the Reproduce section, I could see this being an issue on a repo like the main TOP one. For this repo I don't think there would be as much need for super detailed steps to reproduce something since a lot of bugs would probably be able to be summed up in a single sentence (e.g. "Click on this link" if a link is broken, "Open this lesson in Chrome" if there's some issue with content rendering in Chrome, etc).

Reply:
That's a fair point. I suppose it makes sense that a vast majority are going to be minor things. We can see how it goes and if it feels like this is an issue it can always be revisited. In that case it looks good otherwise! 👍